src/components/[guild]/JoinModal/hooks/useConnectPlatform.ts

```diff
@@ -63,6 +61,10 @@ const useConnectPlatform = (
     onSuccess: () => {
       addDatadogAction("Successfully connected 3rd party account")
       mutateUser()
+      toast({
```

Comment:
this shouldn't be abstracted here, but passed as onSuccess in LinkMoreSocialAccount, like in other components that uses the hook (eg ConnectRequirementPlatformButton). So we can have different toasts for different cases (now it shows double toasts in other cases)